backend/tests/apps/mentorship/api/internal/nodes/test_api_internal_mentor.py

```
@@ -0,0 +1,68 @@
+    """Test name when no github_user is associated."""
+    assert mock_mentor_node_no_github_user.name() == ""
+
+
+def test_mentor_node_login(mock_mentor_node):
+    """Test the login field resolver."""
+    assert mock_mentor_node.login() == "mentorlogin"
+
+
+def test_mentor_node_login_no_github_user(mock_mentor_node_no_github_user):
+    """Test login when no github_user is associated."""
+    assert mock_mentor_node_no_github_user.login() == ""
```

Ruff (0.14.10)

No newline at end of file

Add trailing newline

(W292)